nativeauthenticator/handlers.py

```diff
@@ -136,7 +136,7 @@ async def post(self):
         if assume_human:
             user_info = {
                 "username": self.get_body_argument("username", strip=False),
-                "signup_password": self.get_body_argument("pw", strip=False),
+                "password": self.get_body_argument("signup_password", strip=False),
```

Comment:
In the call to create_user, doesnt this name go onwards as a kwargs, and end up influencing the UserInfo object? We cant change the variable names on that.

Reply:
I guess it would. Sorry, my mistake.